### ISSUE TITLE
TextField with returnKeyType next is not updating the ScrollView position on returnPressEvent

### ISSUE THREAD

**Issue by NickIliev** [NativeScript/NativeScript#5086]
When we have multiple TextField elements inside a ScrollView, the returnPress event is not updating the ScrollView position. 

However, the scroll view position will be updated if the user starts typing into the selected TextField.
Reproducible on both Android and iOS.

Steps to reproduce:
- use [this sample application](https://github.com/NickIliev/NS-Issues/tree/master/others/t_1140109)
- or use the following snippet
```
<ScrollView>
    <StackLayout class="p-20">
        <Label text="11" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="22" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="33" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="44" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="55" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="66" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="77" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="88" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="99" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="1010" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="1111" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
        <Label text="1212" textWrap="true" />
        <TextField returnKeyType="next" hint="some" />
    </StackLayout>
</ScrollView>
```

- start typing in the first few text fields and then go to the next one using the `next` button on the keyboard.

Result: when the focus is in a text field which is behind the keyboard the scroll-view won't update its position, and the selected text-field will remain hidden behind the keyboard.
Only after the user starts typing the scroll position will be updated, and the text field will be visible.

**Note:** to trigger the `returnKeyType` with `next` make sure you are using the tns-core-modules [with this fix applied](https://github.com/NativeScript/NativeScript/pull/5047/files/789b53d3aab640fb0f48090d2b254d6823dc2691)

**Comment by NickIliev** [comment]
Very good detailed description of the issue with ScrollView and the keyboard overlapping the TextFields can be found [here](https://github.com/NativeScript/NativeScript/issues/5088)

**Comment by Chanon-SVN** [comment]
I have this problem too.

**Comment by ajkendall** [comment]
I have this issue as well

**Comment by abner** [comment]
I have this issue too.

**Comment by NickIliev** [comment]
+++ reported [here](https://github.com/NativeScript/NativeScript/issues/1247#issuecomment-364410051)

**Comment by NickIliev** [comment]
Fix confirmed and tested with official (4.0.0) - demo application [here](https://github.com/NickIliev/NS-Issues-2018/tree/master/NativeScript/issue_1247)
